pyefis.xml

```diff
@@ -1,147 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
OK I just wanted to verify.


On 06/15/16 07:22, Phil Birkelbach wrote:
>
> In pyefis.xml 
> <https://github.com/makerplane/pyEfis/pull/7#discussion_r67143788>:
>
> > @@ -1,147 +0,0 @@
> > -<?xml version="1.0"?>
>
> That's correct. I renamed it and changed it a bit but it's essentially 
> the same file.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/makerplane/pyEfis/pull/7/files/b7d331092d2ba33e63fc84973846ebc64c62591d..ae862f5d8f7186215d66498f1af051eac4c202cc#r67143788>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AEjkWUI_BAfSqIRE21FC3By5yrhmMAbyks5qL-BngaJpZM4IwHg1>.
>